spicelib/sim/sim_runner.py

```diff
@@ -382,15 +394,15 @@ def run(self, netlist: Union[str, Path, BaseEditor], *, wait_resource: bool = Tr
                 self.active_tasks.append(t)
                 t.start()
                 sleep(0.01)  # Give slack for the thread to start
-                return t  # Returns the task object
+                return t.raw_file, t.log_file  # Returns the raw and log file
```

Comment:
This is problematic. We need this on other parts of the code, such as the Client Server. 
The way it worked before, is that the SimRunner iterator would return the proper name of the raw and log files. This gave the user all he needed.

Reply:
Ok.
I just wanted to let you know that you have shown another way to solve my need in the example file.
So my previous change is not necessary.